tests/migrations/test_operations.py

```diff
@@ -51,7 +51,7 @@ def make_test_state(self, app_label, operation, **kwargs):
         return project_state, new_state
 
     def set_up_test_model(
-            self, app_label, second_model=False, third_model=False,
+            self, app_label, second_model=False, third_model=False, index=False,
```

Comment:
I seems to me you might have misunderstood my intention here. I would like to have a test for a single column index as well as one for multi column. Your former test / code is here: https://github.com/django/django/pull/6726/files/f31d876fdbdd205e9e49d1af6afcd9f727aecdbd#r66766258